dali/imgcodec/decoders/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+add_subdirectory(jpeg)
```

Comment:
Hmm... so now it only works because we have `-fvisibility=hidden`, doesn't it?

Reply:
I guess so